celerdata/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "790d8932-0833-43ac-b9d8-d6d0a4f11517",
+  "app_id": "celerdata",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gathers CelerData metrics and logs",
+    "title": "CelerData",
+    "media": [],
```

Comment:
Images are required for integrations and will show on the tile. Please include images of graphs, the out of the box dashboard, etc.

Reply:
‌‌Currently, we do not plan to provide a dashboard, so can we provide a screenshot of the metric summary instead?

Reply:
Dashboards are required for integrations. Please provide a dashboard using our best practices [here](https://datadoghq.dev/integrations-core/guidelines/dashboards/)

Reply:
Please include 3 images for your tile. This can include images of graphs, logs, the out of the box dashboard, etc.